site/public/js/forum.js

```diff
@@ -985,6 +994,7 @@ function dynamicScrollLoadPage(element, atEnd) {
         load_page_callback = function(content, count) {
             spinner_down.hide();
             arrow_down.before(content);
+            // eslint-disable-next-line eqeqeq
             if (count == 0) {
```

Comment:
The same behavior was produced when scrolling through the discussion forum posts after replacing `count == 0` with `count === 0`.  When getting the type of `count`, `number` is returned.